Package@swift-5.5.swift

```diff
@@ -30,7 +30,8 @@ let package = Package(
                 "CAtomic",
                 .product(name: "cmark-gfm", package: cmarkPackageName),
                 .product(name: "cmark-gfm-extensions", package: cmarkPackageName),
-            ]),
+            ],
+            exclude: ["Markdown.docc"]),
```

Comment:
The DocC catalog needs to among the target's sources to be associated with the target and to be used as input when building documentation for the target. 

Here in the Swift 5.5. package manifest, where DocC catalogs are supported, "Markdown.docc" shouldn't be excluded.